scripts/upgrade/jellyfin.sh

```diff
@@ -1,68 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
could just leave an `echo "Please use apt to upgrade jellyfin"` but that's cosmetics only

Reply:
What is this referencing?

Reply:
The file in its entirety.

Reply:
The only part that will update is this, the rest is migration.

https://github.com/userdocs/swizzin/blob/master/scripts/update/jellyfin.sh#L46#L79

Does this mean the apt-get will be carried out twice. once by a function and again by this script?

Reply:
nono I meant that if a user attempts to run `box upgrade jellyfin` they're told to use apt instead and nothing else

Reply:
i feel like that is an extra step. as a user i'd prefer if the command gave me the result i expected. to update/fix jellyfin. we are already using build in functions to do the apt-get